model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/README.md

```diff
@@ -1,58 +1,53 @@
+Several Keras inferable quantizers were implemented for activation quantization:
+```markdown
+ActivationPOTInferableQuantizer
+ActivationSymmetricInferableQuantizer
+ActivationUniformInferableQuantizer
+```
+Each of them should be used according to the quantization method of the quantizer (power-of-two, symmetric and uniform quantization respectively).
 
-Another abstract method that should be implemented is `get_config` which returns a dictionary of the arguments that should be passed to the quantizer `__init__` method.
+Similarly, several Keras inferable quantizers were implemented for weights quantization:
+```markdown
+WeightsPOTInferableQuantizer
+WeightsSymmetricInferableQuantizer
+WeightsUniformInferableQuantizer
+```
+Each of them should be used according to the quantization method of the quantizer (power-of-two, symmetric and uniform quantization respectively).
 
-For example:
+## Usage Example
 
 ```python
+# Import TensorFlow and quantizers_infrastructure
 import tensorflow as tf
-from model_compression_toolkit import quantizers_infrastructure as qi
-
-
-class MyQuantizer(qi.BaseKerasInferableQuantizer):
-    def __init__(self,
-                 quantization_target: qi.QuantizationTarget):
-        super(MyQuantizer, self).__init__(quantization_target=quantization_target)
-
-    def get_config(self):
-        return {'quantization_target': self.quantization_target}
-
-    def __call__(self, inputs: tf.Tensor):
-        # Your quantization logic here
-        return tf.round(inputs)
 
+from model_compression_toolkit import quantizers_infrastructure as qi
 
-quantization_target = qi.QuantizationTarget.Activation
-quantizer = MyQuantizer(quantization_target=quantization_target)
-input_tensor = tf.random.normal(shape=(1, 10))
+# Create a weights symmetric quantizer for quantizing a kernel using 8 bits,
+# quantization per-channel (the channel index is passed as channel_axis, when -1 is the last axis),
+# 3 threshold (in this example, the layer's kernel outputs 3 channels, thus we're
+# using 3 thresholds - threshold per channel) and the quantization is signed.
+quantizer = qi.keras_inferable_quantizers.WeightsSymmetricInferableQuantizer(num_bits=8,
+                                                                             threshold=[2, 4, 1],
+                                                                             per_channel=True,
+                                                                             channel_axis=-1,
+                                                                             signed=True)
+
+# Initialize a random input qo quantize
+input_tensor = tf.random.uniform(shape=(1, 3, 3, 3), minval=-100, maxval=100)
+# Quantize tensor
 quantized_tensor = quantizer(input_tensor)
 print(quantized_tensor)
-```
+assert tf.reduce_max(quantized_tensor) < 4, f'Quantized values should not contain values greater than maximal threshold'
```

Comment:
Why the user need this assert?

Reply:
It does not need it, but it helps to understand what we expect to be the output

Reply:
So maybe expain what the expectation after this code section.